Makefile

```diff
@@ -62,9 +62,18 @@ oci-validate-json: validate.go
 oci-validate-examples: cmd/oci-validate-examples/main.go
 	go build ./cmd/oci-validate-examples
```

Comment:
@stevvooe moved `oci-validate-examples` as a unit test, PTAL https://github.com/opencontainers/image-spec/pull/65/commits/71e594293d741708c37a13331edafb8aec6a6ff0. I did not do big refactorings, just a minimalistic 1:1 transition.